### PR TITLE
 Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by Danny-imo**
### **In This Milestone:**
- Build/Complete mobile version of  a personal portfolio site.
- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements in the page.
- Use CSS Grid to place elements in the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,5 +59,157 @@ <h2 class="social">LET'S CONNECT</h2>
     </ul>
 </div>
 </section>
+<section id="About">
```
Comment (braincee) [inline index.html:62]: - [ ] Please I highly recommend wrapping the main work section container with the CSS property `display: grid`  which implements mobile responsiveness, a better user experience per the project requirements. 😄

File: index.html [hunk 2]
```diff
@@ -55,5 +59,157 @@ <h2 class="social">LET'S CONNECT</h2>
...
+        <p class="canopy-txt">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required
+        </p>
+        <ul class="tech1">
+          <li class="tech-t">HTML</li>
+          <li class="tech-t">CSS</li>
+          <li class="tech-j">JavaScript</li>
+        </ul>
+        <button type="button" class="project-btn">See Project</button>
```
Comment (braincee) [inline index.html:81]: - [ ] Please I highly suggest implementing hover effects to the button:  `Active`, `Enabled`, `Hover` and `Pressed` per [Figma Design Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=Tg5tjOlgyejotrXq-0). This effect should be implemented on all buttons in the project. 😄 

_This is a screenshot of the button effects required by the Figma Design_

Screenshot (321)

File: style.css [hunk 3]
```diff
@@ -79,3 +79,293 @@ a {
 .ham {
   list-style-type: none;
 }
+
+/* card section */
+
```
Comment (braincee) [inline style.css]: - [ ] Please I suggest implementing CSS media queries with a minimum `375px` for mobile screen sizes. The whole content of the project should fit the minimum requirement.  😄 

- [ ] This is a video to demonstrate how the whole content of your project displays on a `375px` mobile screen size. 😄 


https://github.com/Danny-imo/my-project/assets/78104544/945ee92c-d0aa-438f-bc41-799c944d7ca1

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @Danny-imo 👋, I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -1,81 +1,387 @@
...
+  .about-section {
+    width: 327px;
+    height: 552px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    padding-top: 15px;
+    margin-top: 114px;
+    margin-left: 15px;
+  }
```
Comment (Alusp) [inline style.css:210]: -[OPTIONAL] In order for you to give a breathing space between the button and the card, Please ensure that you add a padding-bottom to this `about-section` class to match up to the design requirement. Below is the screenshot of your current design.

image

Comment (Alusp) [reply style.css:210]: - [OPTIONAL] Please ensure that the display properties is set to `display:flex` as this is part of the requirement. Below is the screenshot: :arrow_down: 

Works Section requirement

File: index.html [hunk 5]
```diff
@@ -55,5 +59,157 @@ <h2 class="social">LET'S CONNECT</h2>
...
+      <li class="pang-item"> <img src="./img/Ellipse 1 (2).png" alt="" class="im"> &nbsp; &nbsp; CSS</li>
+    </ul>
+  </div>
+
+  <div class="nrop">
+    <p>Frameworks <i class="fas fa-angle-right"></i> </p>
+    <hr>
+    <p>Skills &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <i class="fas fa-angle-right"></i></p>
+    <hr>
+  </div>
```
Comment (Alusp) [inline index.html:212]: - [x] [OPTIONAL] According to the project requirement, Please ensure that the  Technologies section use `<ul>` tag and the `<li>` tags. They are supposed to be list items. Below is the screenshot of the requirement.

image


Please ensure that you follow the requirement accordingly. :slightly_smiling_face:

**Review by Alusp — APPROVED** [review]
Hi @Danny-imo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights

- [ ] GitHub flow rightfully used :white_check_mark:
- [ ] Meaningful title and summary for PR :white_check_mark:
- [ ] No linter errors :white_check_mark:
- [ ] Professional Readme used :white_check_mark: 
- [ ] Good commit history :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**